packages/playground-examples/copy/ja/TypeScript/Primitives/Unknown and Never.ts

```diff
@@ -17,75 +17,73 @@ const myAccount = jsonParser(`{ "name": "Dorothea" }`);
 myAccount.name;
 myAccount.email;
 
-// If you hover on jsonParser, you can see that it has the
-// return type of any, so then does myAccount. It's possible
-// to fix this with Generics - but it's also possible to fix
-// this with unknown.
+// 変数jsonParserにカーソルを合わせると、戻り値の型がany型であることがわかります。
+// そして、変数myAccountも同様にany型になります。
+// こちらについて、ジェネリクスを使って型付けすることもできますが、
+// unknown型を使うことによっても修正できます。
```

Comment:
上のfixを型付けにしているのでこちらも型付けでよいかと
```suggestion
// unknown型を使うことによっても型付けできます。
```